lib/unrar2/windows.py

```diff
@@ -62,145 +80,160 @@
 UCM_PROCESSDATA = 1
 UCM_NEEDPASSWORD = 2
 
-architecture_bits = ctypes.sizeof(ctypes.c_voidp)*8
+architecture_bits = ctypes.sizeof(ctypes.c_voidp) * 8
 dll_name = "unrar.dll"
 if architecture_bits == 64:
     dll_name = "x64\\unrar64.dll"
 
-volume_naming1 = re.compile("\.r([0-9]{2})$")
-volume_naming2 = re.compile("\.([0-9]{3}).rar$")
-volume_naming3 = re.compile("\.part([0-9]+).rar$")
+volume_naming1 = re.compile("[.]r([0-9]{2})$")
+volume_naming2 = re.compile("[.]([0-9]{3})[.]rar$")
+volume_naming3 = re.compile("[.]part([0-9]+)[.]rar$")
 
 try:
-    dll_filename = os.path.join(os.path.split(__file__)[0], 'UnRARDLL', dll_name)
-
-    if sys.version_info[:3] == (2, 7, 13):
-        # http://bugs.python.org/issue29082
```

Comment:
Pretty sure we still need these fixes.
https://github.com/pymedusa/Medusa/pull/1881

The fact that this library is modified was listed on [the readme file](https://github.com/pymedusa/Medusa/blob/develop/lib/readme.md), last entry.

Reply:
@sharkykh fixed